### ISSUE TITLE
param builder  drops items

### ISSUE THREAD

**Issue by SimonCropp** [jehna/ga-lite#22]
this https://github.com/jehna/ga-lite/blob/master/src/ga-lite.js#L74

```
var paramsStr = '';
  for (var key in params) {
   paramsStr = '&' + key + '=' + encodeURIComponent(params[key]);
  }
```

should be 

```
var paramsStr = '';
  for (var key in params) {
   paramsStr += '&' + key + '=' + encodeURIComponent(params[key]);
  }
```

note the `+=` concatenation 

also fixed in here https://github.com/jehna/ga-lite/pull/21

**Comment by jehna** [comment]
Thank you for pointing this out. I cherry-picked the commit 389cf74 from your pull request to fix this issue directly

**Comment by jehna** [comment]
Fixed by the pull request #23

**Comment by SimonCropp** [comment]
> Thank you for pointing this out. I cherry-picked the commit

no worries. that other PR needs more discussion anyway
